pkg/DWSUtility.py

```diff
@@ -368,14 +368,14 @@ def do_resource_list(self):
 
             try:
                 crd_obj = self.dws.get_custom_resource_definition(crd)
-            except:
+            except Exception:
```

Comment:
Should use `DWSError` here. And we don't care if it fails?

Reply:
This and others are correcting a Flake8 error (linter). DWSError is a type of Exception

Reply:
Doesn't flake8 complain that this is generic Exception handling? That's why I pointed this out. https://www.flake8rules.com/rules/E722.html

`except:` and `except Exception:` are equivalent in my mind, but maybe flake8 is ok with this.

Reply:
Well the warning went away - so I went with it.